cypress/support/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+  interface Chainable<Subject> {
+    findByPlaceholder(placeholder: string): Chainable<any>
+    assertPageURL(url: string): Chainable<any>
+    assertAuthorized(username: string): Chainable<any>
+  }
+}
+ 
```

Comment:
Add an empty line to the end of the file (maybe, there are spaces in line 10 now, please remove them)